flake.nix

```diff
@@ -20,7 +20,6 @@
 
       treefmtEval = eachSystem (pkgs: treefmt-nix.lib.evalModule pkgs {
         projectRootFile = ".git/config";
-        programs.ormolu.enable = true;
```

Comment:
This shouldn't be removed.

Reply:
if it doesn't work on x86_64-darwin, it could be also only disabled there.